target.go

```diff
@@ -301,6 +301,30 @@ func getGopath() string {
 	return filepath.Join(home, "go")
 }
 
+func getPackageRoot() (string, error) {
```

Comment:
In the meantime, I wrote my own not realizing that you already had one here:
https://github.com/tinygo-org/tinygo/blob/421ef04efb87889d73d5b404202a655f41d2e1e4/target.go#L319
I think it would be best to use that one instead.

Reply:
Oh nevermind, `getPackageRoot` does something different.

Reply:
I will remove getPackageRoot and use "."